Author: Yves-Alexis Perez <corsac@debian.org>
Author: Lionel Le Folgoc <mrpouit@gmail.com>
Description: adapt config to Debian
Especially:

* use X11 session wraper
* start on vt7
* don't show userlist
* disable DMCP

=== modified file 'data/lightdm.conf'
Index: lightdm-0.4.0/data/lightdm.conf
===================================================================
--- lightdm-0.4.0.orig/data/lightdm.conf	2011-06-27 18:43:17.805498735 +0200
+++ lightdm-0.4.0/data/lightdm.conf	2011-06-27 18:43:43.112623806 +0200
@@ -18,7 +18,7 @@
 [LightDM]
 #xserver=default-xserver
 #default-xserver-command=
-#authorization-directory=/var/cache/lightdm/authority
+authorization-directory=/var/cache/lightdm/authority
 #log-directory=/var/log/lightdm
 #theme-directory=/usr/share/lightdm/themes
 #theme-engine-directory=/usr/libexec/lightdm
@@ -26,8 +26,8 @@
 #cache-directory=/var/cache/lightdm
 #xsessions-directory=/usr/share/xsessions
 #default-xsession=
-#session-wrapper=/etc/X11/Xsession
-#minimum-vt=7
+session-wrapper=/etc/X11/Xsession
+minimum-vt=7
 seats=seat-0
 
 #
@@ -78,7 +78,7 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserManager]
-#load-users=true
+load-users=false
 #minimum-uid=500
 #hidden-users=nobody nobody4 noaccess
 #hidden-shells=/bin/false /usr/sbin/nologin
@@ -108,6 +108,6 @@
 # it can be a word and the first 7 characters are used as the key.
 #
 [xdmcp]
-#enabled=true
+enabled=false
 #port=177
 #key=0x0123456789ABCD
